### PR TITLE
Fix netty dependency and do not ship jars with python 

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

**Comment by eddyxu** [comment]
Address #57

File: build.sbt [hunk 1]
```diff
@@ -4,25 +4,33 @@ name := "rikai"
 
 organization := "ai.eto"
 homepage := Some(url("https://github.com/eto-ai/rikai"))
-scmInfo := Some(ScmInfo(url("https://github.com/eto-ai/rikai"),
-                        "git@github.com:eto-ai/rikai.git"))
+scmInfo := Some(
```
Comment (eddyxu) [inline build.sbt:7]: These were auto-formated by vscode

File: python/tests/conftest.py [hunk 2]
```diff
@@ -12,28 +12,17 @@
...
-    jar_dir = Path(rikai.__file__).parent / "jars"
-    jars = [
-        (jar_dir / jar_file).as_posix()
-        for jar_file in jar_dir.iterdir()
-        if jar_file.suffix == ".jar"
-    ]
     return (
         SparkSession.builder.appName("spark-test")
-        .config("spark.jars", ":".join(jars))
+        .config("spark.jars.packages", "ai.eto:rikai_2.12:0.0.1-SNAPSHOT")
```
Comment (eddyxu) [inline python/tests/conftest.py:25]: will later consolidate `SparkTestCase` to use `pytest` style

File: python/rikai/testing/spark.py [hunk 3]
```diff
@@ -36,20 +34,9 @@ class SparkTestCase(unittest.TestCase):
...
-            raise ValueError(
-                f"Rikai Jar is not found on {jar_dir}, "
-                "please run 'sbt package' first"
-            )
-        jars = ":".join(rikai_jar)
-        logger.info("loading jars for spark testing: %s", jars)
         cls.spark = (
             SparkSession.builder.appName("spark-test")
-            .config("spark.jars", jars)
+            .config("spark.jars.packages", "ai.eto:rikai_2.12:0.0.1-SNAPSHOT")
```
Comment (changhiskhan) [inline python/rikai/testing/spark.py:39]: not a release blocker per se but how do you think we should keep the versions sync'd?

**Review by changhiskhan — APPROVED** [review]

**Pull request merged**